### DISCUSSION TITLE
How do I set build type for Debug/Release?

### DISCUSSION THREAD

**Discussion by BMBurstein** [microsoft/vscode-cmake-tools#2398] [Q&A] [answered]
How do I set build type for Debug/Release?

**Comment by bobbrow** [comment, marked as answer]
You can either invoke the `CMake: Select Variant` command, or click on the currently set type in the status bar 
![image](https://user-images.githubusercontent.com/12818240/155388492-f12333bc-c38e-4027-b64f-751ea3248d6e.png)

Either method will pop up a drop down list that will let you select the build type you want.
